src/components/HomePage/SearchResultsArea/Offer/OfferInteractionOptions.js

```diff
@@ -45,7 +48,12 @@ const OfferVisibilityOptions = ({
         }
     };
 
-    if (!loading && (sessionData?.isAdmin || sessionData?.company?._id === offer.owner))
+    const handleApplyURLRedirect = () => {
+        setClosedRedirectDialog();
+        window.open(offer.applyURL, "_blank", "noopener,noreferrer");
```

Comment:
I think when we started thinking about this feature, we talked about telling the company that the request came from us. @imnotteixeira 

Not sure if we have some secutiry concerns by removing the `noreferrer` tag
- https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/noreferrer
- https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referer

Reply:
I think it's fine to remove the `noreferrer` tag, the website can only know where the visitors came from, they don't have access to anything else, as far as I know.

I think the companies will use a special URL to know users come from NIJobs but I'm not sure, so we should allows for the use of the Referer header